runbotics/runbotics-orchestrator-ui/src/src-app/layouts/MainLayout/TopBar/Account.tsx

```diff
@@ -61,7 +61,7 @@ const Account: FC = () => {
     const [isOpen, setOpen] = useState(false);
     const dispatch = useDispatch();
     const hasAdminAccess = useRole([Role.ROLE_ADMIN]);
-    const hasUserReadAccess = useFeatureKey([FeatureKey.TENANT_READ_USER, FeatureKey.USERS_PAGE_READ], { oneOf: true });
+    const hasUserReadAccess = useFeatureKey([FeatureKey.TENANT_READ_USER, FeatureKey.MANAGE_INACTIVE_USERS], { oneOf: true });
```

Comment:
What do you think about renaming this variable to something like hasUsersPageAccess to make it super clear it is used for rendering Users in menu option? I mean userReadAccess suggests that you have access to see all users while feature key MANAGE_INACTIVE_USERS allows only to inactive users tab.